starlite/middleware/authentication.py

```diff
@@ -43,47 +42,34 @@ class AbstractAuthenticationMiddleware(ABC, MiddlewareProtocol):
     The status code to for websocket authentication errors.
```

Comment:
Should we try and support these still or just remove them and make a breaking change?

```python
    error_response_media_type = MediaType.JSON
    """
    The 'Content-Type' to use for error responses.
    """
    websocket_error_status_code = 4000
    """
    The status code to for websocket authentication errors.
    """
```

Reply:
So for websocket connections, also @cofin, we have to handle it here - websockets do not have a response instance and are not treated by error handlers. 

The `error_response_media_type` is ok to drop in my opinion.

Reply:
Lets drop these.